LeaderboardBackend/Controllers/Requests/PasswordAttribute.cs

```diff
@@ -0,0 +1,65 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace LeaderboardBackend.Controllers.Requests
+{
+	public class PasswordAttribute : ValidationAttribute
+	{
+		private static readonly int MIN = 8;
+		private static readonly int MAX = 80;
+
+		public string GetErrorMessageLength(bool tooShort)
```

Comment:
These methods are only used in this class in one place. I think either this methods should be made private and put at the bottom, or they should just be inlined to the `IsValid` method. Up to you, if it were me I'd inline them and use the simplification from the other comment, but I'll let you decide.

Reply:
Yeah I'm just consolidating this into a `GetErrorMessage` method.